molecule/Dockerfile-ubuntu-java11.j2

```diff
@@ -2,7 +2,8 @@ FROM {{ item.image }}
 
 RUN mkdir -p /usr/share/man/man1 && \
     apt-get update -y && \
-    apt-get install -y openjdk-11-jdk wget gnupg
+    apt-get install -y openjdk-11-jdk wget gnupg  && \
+    apt-get install -y dirmngr
```

Comment:
whats the reason for adding dirmgr package ?

Reply:
Docker file Dockerfile-ubuntu-java11.j2 and Dockerfile-ubuntu2004-java11.j2 were exactly same apart from this dirmgr package, so added this in one docker file and deleted other to create no impact.